Lib/selectors.py

```diff
@@ -550,7 +597,38 @@ def unregister(self, fileobj):
                     pass
             return key
 
+        def modify(self, fileobj, events, data=None):
+            """Change a registered file object monitored events or attached data.
+
+            Parameters:
+            fileobj -- file object or file descriptor
+            events  -- events to monitor (bitwise mask of EVENT_READ|EVENT_WRITE)
+            data    -- attached data
+
+            Returns:
+            SelectorKey instance
+
+            Raises:
+            Anything that unregister() or register() raises
+            """
+            return super().modify(fileobj, events, data)
+
         def select(self, timeout=None):
+            """Perform the actual selection, until some monitored file objects are
```

Comment:
You are right, but all I did was copy the original docstring from the abstract base class before editing it. I feel that if I change this line, I should also change the original doc string of the abstract base class.

Then there are a whole bunch of other lines exceeding the maximum line length. I feel that if I change one of them, I should fix them all.